### PR TITLE
D-stake gentx and peer

### PR DESCRIPTION
**Pull request by donadel01**
D-stake
https://www.d-stake.xyz/
donadel#7181
neutronvaloper1npv2wy7vvfkgr2vr05afe3jq0qtyqwlne02tvx

### CODE REVIEW

File: quark/gentxs/gentx-ee4ab1f713ab01e6b2e4a852ecd2cef459fa4fc7.json [hunk 1]
```diff
@@ -0,0 +1 @@
...ateValidator","description":{"moniker":"D-stake","identity":"21A67C27F1A9CD69","website":"https://www.d-stake.xyz","security_contact":"","details":"D-stake is an experienced PoS validator. You can safely stake your funds"},"commission":{"rate":"0.050000000000000000","max_rate":"0.300000000000000000","max_change_rate":"0.050000000000000000"},"min_self_delegation":"1","delegator_address":"neutron1npv2wy7vvfkgr2vr05afe3jq0qtyqwlnrjnj2z","validator_address":"neutronvaloper1npv2wy7vvfkgr2vr05afe3jq0qtyqwlne02tvx","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"9hQxmi3rdFUWnAhVDoHgwXyWrcCl0wCMpkySSuL52LA="},"value":{"denom":"untrn","amount":"1000000000"}}],"memo":"ee4ab1f713ab01e6b2e4a852ecd2cef459fa4fc7@209.145.62.101:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AxK3LGtIHlp0+MPc4VUZ59/aipwKuD8IpP3yjn10KiQI"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["iD0IKJ7OBBVC6XeXkq2h2aY9JSPldqmM9k5M72N/udR4PaPhoeZbevu3GYyX9l19G2bjzwoKzs3spKtdn+1LtA=="]}
```
Comment (albertandrejev) [inline quark/gentxs/gentx-ee4ab1f713ab01e6b2e4a852ecd2cef459fa4fc7.json:1]: Please update `max_change_rate` and `max_rate` in accordance with documentation.

**Review by albertandrejev — CHANGES_REQUESTED** [review]

File: quark/gentxs/gentx-ee4ab1f713ab01e6b2e4a852ecd2cef459fa4fc7.json (hunk 1 above)
Comment (donadel01) [reply quark/gentxs/gentx-ee4ab1f713ab01e6b2e4a852ecd2cef459fa4fc7.json]: @albertandrejev  updated

**Review by albertandrejev — APPROVED** [review]

**Comment by Medniyy** [comment]
Hey there! Some additional details should be provided in the description!
Check this:

https://github.com/neutron-org/testnets/tree/main/quark#fill-pull-request-description

**Comment by donadel01** [comment]
D-stake
https://www.d-stake.xyz
donadel#7181
neutronvaloper1npv2wy7vvfkgr2vr05afe3jq0qtyqwlne02tvx

**Pull request merged**